samples/sample_hevc_fei/src/hevc_fei_encode.cpp

```diff
@@ -238,6 +315,12 @@ mfxStatus FEI_Encode::EncodeFrame(mfxFrameSurface1* pSurf)
         if (MFX_ERR_NONE <= sts && m_syncPoint) // ignore warnings if output is available
         {
             sts = SyncOperation();
+
+            if ((MFX_ERR_NONE == sts) && m_pFile_repack_stat.get())
```

Comment:
Maybe it's better to move this block in SyncOperation wrapper?

Reply:
I prefer to keep this block in EncodeFrame, because in this way it is in the same function as SetRepackCtrl is, to let code viewer get the SetRepackCtrl/GetRepackStat relationship easier. How do you think about it?

Reply:
It makes sense. But I'm afraid it will be a too big function if we always write here all blocks which dump output files.  Let's leave as is for now, but it's better to refactor and move sync operation here and move dumping all output files in separate logical block (i.e. function)